src/MorphTransition.js

```diff
@@ -36,8 +36,9 @@ class MorphTransition extends React.Component {
     render() {
         var width = this.props.width;
         var height = this.props.height;
+        var className = this.props.className;
         return (
-            <svg viewBox={this.viewBox} preserveAspectRatio="xMidYMid meet" fit width={width} height={height}>
+            <svg viewBox={this.viewBox} preserveAspectRatio="xMidYMid meet" fit width={width} height={height} className={className}>
```

Comment:
Yes let's pass all props, in native version it's already like that
https://github.com/gorangajic/react-svg-morph/blob/master/src/MorphTransitionNative.js#L15